spec/controllers/verify/finance_controller_spec.rb

```diff
@@ -142,7 +142,7 @@
 
         expect(@analytics).to have_received(:track_event).with(
           Analytics::IDV_FINANCE_CONFIRMATION, result
-        )
+        ).twice
```

Comment:
@monfresh I'd love your feedback here

To keep the changeset as small as possible, I had the two form submissions use the same event name which resulted in hacks like this `expect(...).twice`.

My guess is that we'll want to split this into two events, one for the form submit and one for the vendor submit? Do you think we should address this now, or in a follow-up PR?